app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsViewModel.kt

```diff
@@ -19,6 +19,7 @@ import io.reactivex.schedulers.Schedulers
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
+// TODO: is this class nessesary?
```

Comment:
I have no idea, if this is the best solution..

Reply:
I think so still, if we want to keep the ticket system / selling

Reply:
```suggestion
```